LABELS.md

```diff
@@ -0,0 +1,48 @@
+# GitHub Labels
+
+Simple Icons uses labels extensively to categorize issues and contributions. This document aims to explain the labels as well as define rules around how labels are used.
+
+## Notes
+
+- We list various time windows here. If you need an extension for your contribution, you can indicate the time you need in the issue or Pull Request subject to the time window. If the request is reasonable, the deadlines in this document are applied **after** the deadline you set for yourself.
+- If you see an issue or Pull Request that exceeded the time windows listed here, feel free to leave a comment reminding the @simple-icons/maintainers to take action.
+
+## Labels
+
+### Good First Issue
+
+The `good first issue` label is intended to make it easier for newcomers to the project to make a first contributions. Issues with this label are expected to not require too much work, allowing newcomers to get familiar with the contribution process without having to worry too much about technical details.
+
+For the first **14 days** after being assigned to an issue, the issue should not be picked up by existing contributors or maintainers. After that, anyone can work on the issue. The `good first issue` label will remain on the issue until it is picked up by someone.
```

Comment:
Not sure if we should explicitly say existing contributors shouldn't pick this up. Specifically, I don't think it's a good idea to tell existing contributors to close a Pull Request to or ask them not to work on an issue because the issue has the `good first issue` label. Rather, I think maintainers should only inform the contributor that such issue are primarily intended for newcomers.

Reply:
I agree! Closing a (potentially) good PR is not a good idea.

Reply:
Not sure if I follow @service-paradis - this paragraph and my comment are both intended to be about issues labeled as `good first issue` that have not been picked up by anyone after 14 days :thinking:

Reply:
I was referring to this part of your comment:

> I don't think it's a good idea to tell existing contributors to close a Pull Request